app/services/s3_zip_downloader.rb

```diff
@@ -7,7 +7,7 @@ def initialize(claim)
 
   def generate!
     "./tmp/#{@claim.case_number}-#{SecureRandom.uuid}-documents.zip".tap do |bundle|
-      build_zip_file @claim.documents, bundle
+      build_zip_file @@claim.documents.includes(:document_blob, :converted_preview_document_attachment), bundle
```

Comment:
One last change, sorry 😬 - I don't think the additional `@` is necessary here. It changes `claim` from an instance variable to a class variable, which isn't needed here (and I think is making your tests fail 😀)